src/Store/SessionStore.php

```diff
@@ -34,12 +39,9 @@ class SessionStore implements StoreInterface
      * @param string|null $base_name      Session base name.
      * @param integer     $cookie_expires Session expiration in seconds; default is 1 week.
      */
-    public function __construct($base_name = null, $cookie_expires = 604800)
+    public function __construct($base_name = self::BASE_NAME, $cookie_expires = self::COOKIE_EXPIRES)
     {
-        if (! empty( $base_name )) {
```

Comment:
What if the user explicitly passes an empty string? You have 2 paths on cases like this:
- be clear on the docs that the base_name can't be null OR
- leave a check like this that always defaults to something valid


I see the docs still mention user could pass null. So this is wrong 🦀

Reply:
I left a check out here because someone, if they really wanted to, could pass null/empty string. There's nothing in the SDK that requires a base name, it's just probably a good idea to do so. Won't hurt anything to not have one.

Reply:
I removed the `null` reference in the docs to push someone towards a string.